analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Conversion.cs

```diff
@@ -26,7 +26,7 @@ internal sealed class Conversion : SimpleProcessor<IConversionOperationWrapper>
         operation.ToConversion();
 
     protected override ProgramState Process(SymbolicContext context, IConversionOperationWrapper conversion) =>
-        context.State[conversion.Operand] is { } value
+        context.State[conversion.Operand] is { } value && conversion.OperatorMethod is null
```

Comment:
Is this the "is user defined conversion" check? If so, it would be better to check [IsUserDefined](https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.operations.commonconversion.isuserdefined?view=roslyn-dotnet-4.3.0#microsoft-codeanalysis-operations-commonconversion-isuserdefined). It is accessible via the [Conversion](https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.operations.iconversionoperation.conversion?view=roslyn-dotnet-4.3.0) property.